api/database/seeders/DatabaseSeeder.php

```diff
@@ -200,13 +200,18 @@ private function truncateTables()
 
     private function seedPoolCandidate(User $user, Pool $pool)
     {
+        $faker = Faker\Factory::create();
+
         // create a pool candidate in the pool
         PoolCandidate::factory()->count(1)->sequence(fn () => [
             'pool_id' => $pool->id,
             'user_id' => $user->id,
-        ])->for($user)->afterCreating(function (PoolCandidate $candidate) {
+        ])->for($user)->afterCreating(function (PoolCandidate $candidate) use ($faker) {
             if ($candidate->submitted_at) {
                 $candidate->createSnapshot();
+
+                $candidate->suspended_at = $faker->optional($weight = 25)->dateTimeBetween('-3 months', 'now');
```

Comment:
I did that initially but it would have caused most of our tests involving pool candidates to need changed to set the field back to null after creating so I went with this. I can move it to PoolCandidateFactory if you think that would be better though.

Reply:
So something like [this](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/727d09b96d222b2d5df7a0ba2b2d2804c5e43bc7)? I'm not sure if it's really an improvement.

Reply:
I like it!  :+1: I hope you don't mind I added a commit with a bit of tidying - [tidy seeder](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5832/commits/76de5a7b38edc5ceadcca8119cf76205b61da3e5).

I'd actually like to move the 
```php
  if ($candidate->submitted_at) {
                    $candidate->createSnapshot();
                }
```
stuff into the factory but I got bunch of lazy load errors that I don't want to deal with right now.  :laughing:

Reply:
Thanks! I hadn't actually looked that closely at the function when I copied it. I think I can tidy it slightly further.

Reply:
Oh, that's really nice!